exercises/hamming/.meta/src/reference/java/Hamming.java

```diff
@@ -1,15 +1,15 @@
-public class Hamming {
+class Hamming {
     private final int hammingDistance;
 
-    public Hamming(String leftStrand, String rightStrand) {
+    Hamming(String leftStrand, String rightStrand) {
         hammingDistance = computeHammingDistance(leftStrand, rightStrand);
     }
 
-    public int getHammingDistance() {
+    int getHammingDistance() {
         return hammingDistance;
     }
 
-    private int computeHammingDistance(String leftStrand, String rightStrand) {
+    int computeHammingDistance(String leftStrand, String rightStrand) {
```

Comment:
This (and the next method) should remain private. We want to be as restrictive as possible with access modifiers and private is more restrictive than package private, see https://docs.oracle.com/javase/tutorial/java/javaOO/accesscontrol.html for an explanation :)

Reply:
@FridaTveit  Thank you for your suggestion. I am just starting to contribute so its really helpful. And I thought that we have to make all the things package private even the private methods. Thank you for the explanation!